mono/metadata/cominterop.c

```diff
@@ -540,6 +541,30 @@ cominterop_com_visible (MonoClass* klass)
 
 }
 
+static guint32
+cominterop_class_iface_type (MonoClass* klass)
+{
+	ERROR_DECL (error);
+	static MonoClass *ClassInterfaceAttribute = NULL;
+	MonoCustomAttrInfo *cinfo;
+	guint32 result = 2;
```

Comment:
What is this magic number, AutoDual? Maybe use a define or something to describe the meaning of the value.

Reply:
Yes, it's ClassInterfaceType.AutoDual. I can add a define.